.github/workflows/docker-build.yaml

```diff
@@ -145,6 +145,11 @@ jobs:
                     registry: ghcr.io
                     username: ${{ github.actor }}
                     password: ${{ secrets.GITHUB_TOKEN }}
+            -   name: Build custom base image if requested
+                if: "contains(github.event.pull_request.labels.*.name, 'build-php')"
```

Comment:
I wanted to distinguish between the "functional labels" and the mere markers. 

But I don't have any strong feelings about the label names.
The suggested one is really long and emphasizes the importance of the label at the expense of others, more frequently used, while in reality this one (and the review-isolated) will be used rarely.

![Snímek obrazovky 2023-09-05 v 13 23 14](https://github.com/shopsys/shopsys/assets/1177414/ccd95af3-3e28-4bed-b815-2b310546023a)

Reply:
Well, at least I can see what it does, as the description is trimmed 😃 

What about `Use PHP Dockerfile instead of built image`? It will be at the end and descriptive.

Reply:
Same then could be used for dependencies :)

Reply:
I have renamed the labels to 
`Use PHP Dockerfile instead of built image` 
`Use docker-compose defined services for review`